src/NzbDrone.Core/Download/TrackedDownloads/TrackedDownloadService.cs

```diff
@@ -278,12 +279,29 @@ public void Handle(EpisodeInfoRefreshedEvent message)
             }
         }
 
+        public void Handle(SeriesAddedEvent message)
+        {
+            var cachedItems = _cache.Values
```

Comment:
Do we need to do the same thing if the cached item doesn't have a series or are those not affected?